libpod/image/pull.go

```diff
@@ -46,6 +46,8 @@ var (
 	AtomicTransport = "atomic"
 	// DefaultTransport is a prefix that we apply to an image name
 	DefaultTransport = DockerTransport
+	// DefaultLocalRepo is the default local repository to pull an image from.
```

Comment:
Should make this more clear that it's just for local image operations and remote pulls will still use our defined registries